extensions/mock-credentials-verifier/build.gradle.kts

```diff
@@ -14,5 +14,6 @@ dependencies {
     testImplementation("org.assertj:assertj-core:${assertj}")
     testImplementation("org.junit.jupiter:junit-jupiter-api:${jupiterVersion}")
     testImplementation("org.mockito:mockito-core:${mockitoVersion}")
+    implementation("${edcGroup}.identityhub:identity-hub-spi:${edcVersion}")
```

Comment:
Variables should be renamed from `edcGroup` and `edcVersion` to `identityHubGroup` and `identityHubVersion`. Specially the versions will deviate at some point.